internal/inspect/r_test.go

```diff
@@ -57,6 +57,17 @@ For more information about these matters see
 https://www.gnu.org/licenses/.
 `
 
+const rOutputWithWarnings = `WARNING: ignoring environment value of R_HOME
+R version 4.3.3 (2024-02-29) -- "Angel Food Cake"
```

Comment:
I know that this is an exact example that we saw in real life, but if we're iterating over lines and want to ensure that we do, in fact, loop over them we should add a few more lines up above this too (and that's a good reminder to future-us that this is not about finding the version in the first or second lines only).